CYS1.py

```diff
@@ -0,0 +1,753 @@
+      (elements C,H,N,O,S)"""
+    elements = ["C", "H", "N", "O", "S"]
+    result = []
+    for seq in seqs:
+        brutto_list = [amino_brutto.get(letter) for letter in seq]
+        brutto_pair = list(zip(*brutto_list))
+        brutto = [sum(i) for i in brutto_pair]
+        brutto_dict = dict(zip(elements, brutto))
+        result.append(brutto_dict)
+    return result
+
+
+def is_length_divisible_by_3(seq: str) -> bool:
+    """
+    Checks if the sequence is divisible by three.
+
+    Arguments:
+      - seq (str): string of protein sequence
+
+      Return:
+      - bool: True if sequence is divisible by three, otherwise False"""
+    seq_len = len(seq)
+    if seq_len % 3 == 0:
+        return True
+    else:
+        return False
+
+
+def is_amino_acid_three_letter(seq: str) -> bool:
+    """
+    Checks whether all elements of a sequence are three-letter
+    amino acid symbols.
+
+    Arguments:
+      - seq (str): string of protein sequence
+
+    Return:
+      - bool: True if sequence is corresponding to the valid
+      three-letter amino acid, otherwise False
+    """
+    seq = seq.lower()
+    seq3 = [seq[i: i + 3] for i in range(0, len(seq), 3)]
+    for triplet in seq3:
+        if triplet not in AMINO_NAMES_DIC.keys():
+            return False
+        else:
+            return True
+
+
+def is_valid_dna_rna(seq: str) -> bool:
+    set_of_seq = set(seq.upper())
+    if set_of_seq > {'A', 'T', 'G', 'C', 'U'}:
+        raise ValueError('Invalid alphabet')
+    elif ('T' in set_of_seq) and ('U' in set_of_seq):
+        raise ValueError('Invalid alphabet')
+    else:
+        return True
+
+
+def transcribe(seq: str) -> str:
+    transcribed_seq = ''
+    for nucleotide in seq:
+        if nucleotide == 'T':
+            transcribed_seq += 'U'
+        elif nucleotide == 't':
+            transcribed_seq += 'u'
+        else:
+            transcribed_seq += nucleotide
+    return transcribed_seq
+
+
+def reverse(seq: str) -> str:
+    reversed_seq = seq[::-1]
+    return reversed_seq
+
+
+def complement(seq: str) -> str:
+    complement_seq = ''
+    dna_complement_map = {
+        'A': 'T',
+        'T': 'A',
+        'G': 'C',
+        'C': 'G',
+        'a': 't',
+        't': 'a',
+        'g': 'c',
+        'c': 'g'
+    }
+    for nucleotide in seq:
+        complement_seq += dna_complement_map[nucleotide]
+    return complement_seq
+
+
+def reverse_complement(seq: str) -> str:
+    return complement(reverse(seq))
+
+
+def get_nucl_acid_type(seq: str) -> str:
+    set_of_seq = set(seq)
+    if ('U' in set_of_seq) or ('u' in set_of_seq):
+        return 'RNA'
+    elif ('T' in set_of_seq) or ('t' in set_of_seq):
+        return 'DNA'
+    return 'ND'
+
+
+class BiologicalSequence(ABC):
+    def __init__(self, sequence):
+        self.sequence = sequence
+
+    def __len__(self) -> int:
+        """Returns the length of the sequence."""
+        return len(self.sequence)
```

Comment:
Круто, что реализовали дандеры и еще и прописали ко всем докстринги! Можно было бы еще добавить __repr__. Но, строго говоря, так как это абстрактный класс, то все эти методы все равно придется прописывать заново, так что тут можно было обойтись просто заглушкой в виде *pass*. Также, необходим декоратор @abstractmethod
```suggestion
    @abstractmethod
    def __len__(self) -> int:
        pass
```